<launch>
   <node pkg="background_foreground_segmentation"
            name="pose_recorder_node"
            type="pose_recorder_node"
            cwd="node"
            output="screen">
        <param name="imageTopic" value="/rgb/bg_fg_img" />
        <param name="cameraFrame" value="/rgb_camera_link" />
        <param name="outputFile" value = "/home/rene/pose_recorder.csv" />
    </node>
</launch>
